src/factories/store/store.factory.ts

```diff
@@ -1,8 +1,14 @@
-import { Rule } from '@angular-devkit/schematics';
-import { StoreSchema } from './store.schema';
-import { FACTORIES } from '../../utils';
+import { Rule, SchematicsException } from '@angular-devkit/schematics';
+
+import { FACTORIES, isEmpty } from '../../utils';
 import { factoryLoader } from '../../utils/factory-loader';
 
+import { StoreSchema } from './store.schema';
+
 export function store(options: StoreSchema): Rule {
+  if (isEmpty(options.name)) {
+    throw new SchematicsException('Invalid options, "name" is required.');
+  }
```

Comment:
```ts
if (isEmpty(options.name)) {
     throw new SchematicsException('Invalid options, "name" is required.');
   }
```

Current code used in two places (action.factory.ts, store.factory.ts), I think this code need moved to factory-loader.ts